materialize-elasticsearch/driver_test.go

```diff
@@ -33,6 +34,9 @@ func testConfig() *config {
 
 func TestValidateAndApply(t *testing.T) {
 	flag.Parse()
+	if os.Getenv("TEST_DATABASE") != "yes" {
```

Comment:
nit: left over from manual testing? Or does this serve a functional role (and if so, add a comment)?